src/styles/navbar.module.scss

```diff
@@ -301,3 +301,11 @@ input:checked + .slider:before {
 .slider.round:before {
   border-radius: 50%;
 }
+
+.account_header {
+  font-weight: bold;
+  color: var(--text-main);
+  :hover {
+    cursor: pointer;
```

Comment:
is hover needed?we are doing for the mobile right

Reply:
no not needed, incase sometime in future if we think of using this on a website. So added it

Reply:
okay

Reply:
made change